src/FakeItEasy/FakeFacade.cs

```diff
@@ -50,6 +51,7 @@ public virtual void ClearRecordedCalls(object fakedObject)
             manager.ClearRecordedCalls();
         }
 
+        [Obsolete("Test fixture initialization will be removed in version 5.0.0.")]

```

Comment:
I was thinking to keep us from starting to use it accidentally. But that _does_ seem unlikely. Removing the attribute.